### PR TITLE
Change path of vector{2|3|4}.rs

### LINKED ISSUE #23: Change Vector{2|3|4} Location
* Change Vector{2|3|4}.rs to parent directory

### PR DESCRIPTION
**Pull request by BeLeap**
This revision includes:
- Change path of vector{2|3|4}

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,5 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+.idea/
```
Comment (utilForever) [inline .gitignore]: Can you add the comment what it is?

**Review by utilForever — CHANGES_REQUESTED** [review]
Thanks for your work! Please check for minor feedback. 😉

File: .gitignore (hunk 1 above)
Comment (BeLeap) [reply .gitignore]: Done

**Review by utilForever — APPROVED** [review]
LGTM now. 🎉

**Pull request merged**